clrtrust

```diff
@@ -39,6 +46,24 @@ detect_c_rehash() {
     return 0
 }
 
+sig_ignore() {
+    :
+}
+
+lock() {
+    # lock should be on the same block device so that we could ln to it
+    local lock_path=$(mktemp -p "$(dirname $CLR_TRUST_LOCK_FILE)")
```

Comment:
It would be slightly simpler I think to say

        local lock_path=$(mktemp "${CLR_TRUST_LOCK_FILE}.xxxxxx")

Reply:
Indeed!